src/app/phase1/phase1.component.ts

```diff
@@ -19,39 +14,20 @@ import { GithubService } from '../core/services/github.service';
 export class Phase1Component implements OnInit {
   issues: BehaviorSubject<Issue[]>;
```

Comment:
Actually, I don't think `issues` attribute is being used. Can remove it if it is no longer in use.